generated/nidaqmx/_grpc_interpreter.py

```diff
@@ -3376,7 +3376,7 @@ def write_digital_u32(
             grpc_types.WriteDigitalU32Request(
                 task=task, num_samps_per_chan=num_samps_per_chan,
                 auto_start=auto_start, timeout=timeout,
-                data_layout_raw=data_layout, write_array=write_array))
+                data_layout_raw=data_layout, write_array=write_array.flat))
         return response.samps_per_chan_written
 
     def write_digital_u8(
```

Comment:
bytes() doesn't handle this case correctly for uint8.
```
>>> import numpy
>>> x = numpy.asarray(True, dtype=bool)
>>> bytes(x)
b'\x01'
>>> y = numpy.asarray(False, dtype=bool)
>>> bytes(y)
b'\x00'
>>> z = numpy.asarray(0xa5, dtype=numpy.uint8)
>>> bytes(z)
b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00'
```

It's interpreting the value 0xa5 (165) as a size:
```
>>> len(bytes(z))
165
```

`x.flat` works, but it reports DeprecationWarning for bool.
```
>>> bytes(x.flat)
<stdin>:1: DeprecationWarning: In future, it will be an error for 'np.bool_' scalars to be interpreted as an index
b'\x01'
>>> bytes(y.flat)
b'\x00'
>>> bytes(z.flat)
b'\xa5'
```

Reply:
Thank you! I have changed to `tobytes()` and verified by running tests for any discrepancies. All expected tests passed :)